prj.conf

```diff
@@ -70,3 +70,14 @@ CONFIG_MEMFAULT_COREDUMP_COLLECT_TASKS_REGIONS=n
 CONFIG_HEAP_MEM_POOL_SIZE=1024
 
 CONFIG_ASSERT=y
+
+# Enable MCUboot
+CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_SHELL=y
```

Comment:
Wow, I guess I was always enabling this in my build command, good addition!

Reply:
Actually I just double checked and `CONFIG_SHELL` is being y-selected by `CONFIG_NET_SHELL=y` so that's how we got it without explicitly enabling. But I think we'll want to enable it anyways since both this shell and `CONFIG_NET_L2_WIFI_SHELL` depend on it. I'm going to move it up to the top with `CONFIG_EARLY_CONSOLE` since multiple configs depend on it